src/Chirp.Core/Interfaces/IAuthorRepository.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Chirp.Core.DTOs;
+
+namespace Chirp.Core.Interfaces;
+
+public interface IAuthorRepository
+{
+    Task CreateAuthor(string authorName, string authorEmail);
+    Task DeleteUser(AuthorDTO Author);
+    Task FollowAuthor(string userAuthor, string followedAuthor);
```

Comment:
Can we keep the camelcase also in the attributes of the methods

Reply:
Marcus have done it the correct way with writing [camel case](https://da.wikipedia.org/wiki/CamelCase). I think you might confuse it with [Pascal Case](https://www.theserverside.com/definition/Pascal-case).
- thisIsCamelCase
- ThisIsPascalCase

Reply:
On that note, the methods should per standard be Camel Case aswell, but that's minor.